db/migrate/20200624162213_add_challenge_end_time.rb

```diff
@@ -0,0 +1,5 @@
+class AddChallengeEndTime < ActiveRecord::Migration[6.0]
+  def change
+    add_column :challenges, :challenge_end, :datetime
```

Comment:
Should be able to just delete this migration and the remove migration since they never made it into master. You may have to `rails db:rollback STEPS=3`, delete the 2 files and then migrate again.